docs/rfcs/0002-storing-course-data.md

```diff
@@ -0,0 +1,134 @@
+## Storing course data
+
+### Abstract
+
+The aim of this RFC is to make decisions about the following:
+- The model definitions* that will be needed to store course data.
+- The plan for loading legacy course data, and keeping that data current
+  after we switch to using our own Open edX instance.
+
+Out of scope for this RFC:
+- Content management – we will just rely on Django admin for creating/updating
+  courses by hand.
+- Any kind of grading/certificate data.
+
+\* - _Much like MicroMasters, we will keep our own models for courses/programs
+  that will exist in parallel with the course data in edX_
+
+### Models
+
+Since this app has very similar needs, `Program`, `Course`, and `CourseRun` 
+models will very closely mirror the [models in MicroMasters](https://github.com/mitodl/micromasters/blob/master/courses/models.py), 
+minus a few fields that are not relevant for this app.
+
+`Program`
+- title
+- description
+- thumbnail (image)
+- live (boolean - indicates whether the program is live in edX, et. al.)
+- merge_key (string - allows us to maintain a reference to a single program
```

Comment:
I don't know what this is or how it helps

Reply:
@Ferdi see [this initial comment](https://github.com/mitodl/mitxpro/pull/23#discussion_r260811766) from @rhysyngsun. Example scenario: we have a Program with 5 Courses. Some user has already finished 2 Courses on the old platform, and takes the next 3 Courses on the new platform. The Program and all of its Courses would have 2 records each in our database (1 for each platform), and we will probably want to be able to maintain a relationship between the Courses taken for the Program on the old and new platforms. `merge_key` would be used to maintain that relationship, and would be more dependable than using something like `title`